pygfx/renderers/wgpu/_mipmapsutil.py

```diff
@@ -54,155 +57,144 @@
 """
 
 
-class MipmapsUtil:
-    def __init__(self, device) -> None:
-        self.device = device
-
-        # Cache pipelines for every texture format used.
-        self.pipelines = {}
+def get_mip_level_count(texture):
+    """Get the number of mipmap levels from the texture size."""
+    assert isinstance(texture, Texture)
+    width, height, _ = texture.size
+    return math.floor(math.log2(max(width, height))) + 1
```

Comment:
why not numpy?

Reply:
This was already written like that. Converted to numpy.